templates/legal/data-privacy/maas-user-survey.md

```diff
@@ -0,0 +1,93 @@
+---
+wrapper_template: "../_base_legal_markdown.html"
+context:
+  title: "Privacy Notice – MAAS User Survey"
+  description: This privacy notice tells you about the information collected from you when you take part in a MAAS user survey on Canonical’s MAAS webpage, hosted by Usabilla.
+  copydoc: https://docs.google.com/document/d/1XKWK3VvJe0Z_soJqO3F0BrrXwlP9ZObYYKxA1hym7QU/edit
+---
+
+# Privacy Notice – Snapcraft NPS Survey  
+
+Version -  December 2018
+
+This privacy notice tells you about the information collected from you when you take part in a MAAS user survey on Canonical’s MAAS webpage, hosted by Usabilla. Usabilla is a third party software provider that enables live user feedback. In collecting this information, Usabilla are acting as a data controller and details of their terms can be found on [their website](https://usabilla.com/terms/). Usabilla will be sharing your data with us and by law, we are required to provide you with information about us, about why and how we use your data, and about the rights you have over your data. For more information, please see our [privacy policy](/legal/data-privacy).
+
+## Who are we?
+
+We are Canonical Group Limited. Our address is 5th Floor Blue Fin Building, 110 Southwark Street, London, SE1 0SU. You can contact us by post at the above address or by email to [dataprotection@canonical.com](mailto:dataprotection@canonical.com) for the attention of “Legal”.
+
+We are not required to have a data protection officer, so any enquiries about our use of your personal data should be addressed to the contact details above.
+
+## What personal data do we collect?
+
+When you submit live feedback on the MAAS webpages, we invite you to leave your your email address and related information via Usabilla’s software platform. You may be shown different surveys depending on user features and usage. Usabilla will share this information with us.
+
+## Why do we collect this information?
+
+We will use your information for MAAS user research. We ask for your consent to do this and will only contact you for this purpose for as long as you continue to consent. This may involve inviting you to talk to us about how you use MAAS and to test new MAAS features.
+
+## What do we do with your information?
+
+Your information is stored in our database and may be processed outside of the European Economic Area. It is shared with the third parties only as reasonably required for Canonical to process and store your data in accordance with this notice.
```

Comment:
The data is not stored in our database. We just have access to the data held by Usabilla.